scripts/build-products.js

```diff
@@ -110,6 +110,7 @@ const fetchProducts = async (coverProducts, provider) => {
         coverAssets: parseProductCoverAssets(coverAssets, coverAssetsMap),
         isPrivate,
         timestamp,
+        minPrice: minPrice || 100, // default to 1% if minPrice is not set
```

Comment:
Create a constant for `100` please. In Cover.sol we call it `DEFAULT_MIN_PRICE_RATIO`

Reply:
Addressed on [0fafea5](https://github.com/NexusMutual/sdk/pull/272/commits/0fafea50cb5f37ed8f1916119a62935a8773dc67). Thx for the feedback 🙏